packages/react/src/components/calendar/CalendarDay.tsx

```diff
@@ -11,7 +10,6 @@ export interface IDay {
     isSelected?: boolean;
     isLowerLimit?: boolean;
     isUpperLimit?: boolean;
-    color?: string;
```

Comment:
The commit will have to be tagged as a breaking change since we are removing something that users of plasma might be currently using in their code 😉 

To do so, you will have to amend the commit and change the message